units/en/unitbonus2/hands-on.mdx

```diff
@@ -0,0 +1,3 @@
+# Hands-on [[hands-on]]
+
+Now that you've learned to use Optuna, **why not going back to our Deep Q-Learning hands-on and implement Optuna to find the best training hyperparameters?**
```

Comment:
Maybe this is a bit hard as we're using systems that require 90 minutes to run in a Colab. Would there would be some way to use the Q Learning Hands On (so not the Deep Q Learning one) or the PPO from Unit 1, with an environment which is slightly more challenging? 

The notebook could be quite simple and just define the environment and configure an initial set of hyperparameters + the train/push code, but then it's on the course participants to do the tuning

Reply:
Likely the PPO from Unit 1 could make lots of sense and it's very similar to the lab from https://colab.research.google.com/github/araffin/tools-for-robotic-rl-icra2022/blob/main/notebooks/optuna_lab.ipynb#scrollTo=TP9C9AqLndxz

Reply:
I will say why not trying with the first Unit 1: Lunar Lander to see if you can increase your score and then also I will let the second phrase for motivated people

Reply:
Changed ✔️